### DISCUSSION TITLE
Waybar Hide

### DISCUSSION THREAD

**Discussion by sayskar500** [prasanthrangan/hyprdots#736] [Q&A] [answered]
Is there any keybinding to hide the taskbar for HyprLand in Arch Linux ? 

I was logging into a site and suddenly the taskbar hid. I am not if it was a bug or not because I don't know how to reproduce it as I don't know how it happened.

**Comment by Senshi111** [comment]
waybar?

**Reply by sayskar500** [reply]
Yes Waybar

**Comment by kRHYME7** [comment, marked as answer]
Hello @sayskar500 

you can run ``` Hyprdots show binds ``` then search for waybar to see some waybar-related actions.  


![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/40c15b21-3547-4eca-aa4b-b28a3ec35061)

**Reply by sayskar500** [reply]
Yeah that helped out. I am new to Hyprland so confused sometimes with a few silly stuff as well. Thanks a lot.

**Comment by kRHYME7** [comment]
No probs! You can try to read the https://wiki.hyprland.org/ for some things but at least in case you're new to all Windows Manager things like Hyprland you should remind yourself that Hyprland is very different with Desktop Environment like KDE or Gnome.Good luck to your venture.

**Reply by sayskar500** [reply]
thanks a lot man ... gonna go through this now more than often ... also this part of the community is way better than arch ones ... the moderators and everyone in general is super toxic (ofc idc but just thanks for the help)
